optional-requirements.txt

```diff
@@ -0,0 +1,6 @@
+pynput==1.7.3
+python-xlib==0.30
+six==1.16.0
+pyobjc==7.3
```

Comment:
Instead of having an optional requirement for macos, we could use `sys_platform` to make a requirement optional/mandatory for a platform. 

Checkout this -> http://stackoverflow.com/a/35614580/872328

Reply:
This way `pip install -r requirements.txt` would work for both windows and linux

Reply:
Sure I'll be doing that